data-prepper-plugins/opensearch/src/test/java/org/opensearch/dataprepper/plugins/sink/opensearch/index/DefaultIndexManagerTests.java

```diff
@@ -100,8 +100,10 @@ public void setup() throws IOException {
     public void getIndexAlias_IndexWithTimePattern(){
         when(indexConfiguration.getIndexAlias()).thenReturn(INDEX_ALIAS_WITH_TIME_PATTERN);
         defaultIndexManager = indexManagerFactory.getIndexManager(IndexType.CUSTOM, restHighLevelClient, openSearchSinkConfiguration);
-        final String indexAlias = defaultIndexManager.getIndexAlias();
-        assertTrue(EXPECTED_INDEX_PATTERN.matcher(indexAlias).matches());
+        try {
+            final String indexAlias = defaultIndexManager.getIndexName(null);
+            assertTrue(EXPECTED_INDEX_PATTERN.matcher(indexAlias).matches());
```

Comment:
Please use `assertThat(string, matchPattern(pattern))`. You use this in another test in this PR.